main.php

```diff
@@ -2,153 +2,166 @@
+	<h2><?php esc_html_e( 'Contact', 'theme-check' ); ?></h2>
+	<p>
+	<?php
+	printf(
+		esc_html__( 'Theme Check is maintained by %1$s and %2$s.', 'theme-check' ),
 		'<a href="https://profiles.wordpress.org/otto42/">Otto42</a>',
 		'<a href="https://profiles.wordpress.org/pross/">Pross</a>'
-		); ?></p>
-	<p><?php printf( __( 'If you have found a bug or would like to make a suggestion or contribution, please leave a post on the <a href="%1$s">WordPress forums</a>, or talk about it with the theme review team on <a href="%2$s">Make WordPress Themes</a> site.', 'theme-check' ), 'https://wordpress.org/tags/theme-check?forum_id=10', 'https://make.wordpress.org/themes/') ; ?></p>
-	<p><?php printf( __( 'The code for Theme Check can be contributed to on <a href="%s">GitHub</a>.', 'theme-check' ), 'https://github.com/Otto42/theme-check'); ?></p>
-	<h3><?php _e( 'Testers', 'theme-check' ); ?></h3>
-	<p><a href="https://make.wordpress.org/themes/"><?php _e( 'The WordPress Theme Review Team', 'theme-check' ); ?></a></p>
+	);
+	?>
+		</p>
+	<p><?php printf( esc_html__( 'If you have found a bug or would like to make a suggestion or contribution, please leave a post on the <a href="%1$s">WordPress forums</a>, or talk about it with the theme review team on <a href="%2$s">Make WordPress Themes</a> site.', 'theme-check' ), 'https://wordpress.org/tags/theme-check?forum_id=10', 'https://make.wordpress.org/themes/' ); ?></p>
+	<p><?php printf( esc_html__( 'The code for Theme Check can be contributed to on <a href="%s">GitHub</a>.', 'theme-check' ), 'https://github.com/Otto42/theme-check' ); ?></p>
+	<h3><?php esc_html_e( 'Testers', 'theme-check' ); ?></h3>
+	<p><a href="https://make.wordpress.org/themes/"><?php esc_html_e( 'The WordPress Theme Review Team', 'theme-check' ); ?></a></p>
 	<?php
 }
 
 function tc_success() {
 	?>
-	<div class="tc-success"><p><?php _e( 'Now that your theme has passed the basic tests you need to check it properly using the test data before you upload it to the WordPress Themes Directory.', 'theme-check' ); ?></p>
-	<p><?php _e( 'Make sure to review the guidelines at <a href="https://make.wordpress.org/themes/handbook/review/required/">Theme Review</a> before uploading a Theme.', 'theme-check' ); ?></p>
-	<h3><?php _e( 'Useful Links', 'theme-check' ); ?></h3>
+	<div class="tc-success"><p><?php esc_html_e( 'Now that your theme has passed the basic tests you need to check it properly using the test data before you upload it to the WordPress Themes Directory.', 'theme-check' ); ?></p>
+	<p><?php esc_html_e( 'Make sure to review the guidelines at <a href="https://make.wordpress.org/themes/handbook/review/required/">Theme Review</a> before uploading a Theme.', 'theme-check' ); ?></p>
```

Comment:
Using esc_html_e here breaks the HTML for the link

Reply:
```suggestion
	<p>
		<?php
		printf(
			/* translators: 1 is an opening anchor tag 2 is the closing part of the tag. */
			esc_html( 'Make sure to review the guidelines at %1$sTheme Review%2$s before uploading a Theme.', 'theme-check' ),
			'<a href="https://make.wordpress.org/themes/handbook/review/required/">',
			'</a>'
		);
		?>
	</p>
```